dockstore-webservice/src/main/resources/migrations.1.15.0.xml

```diff
@@ -29,4 +29,28 @@
         <dropPrimaryKey tableName="user_entry"/>
         <addPrimaryKey columnNames="userid, entryid" constraintName="user_entry_pkey" tableName="user_entry"/>
     </changeSet>
+    <changeSet id="unique_checker_ids" author="coverbeck">
+        <sql dbms="postgresql">
+            UPDATE tool SET checkerid=NULL where checkerid in (SELECT checkerid from workflow);
```

Comment:
Not sure I understand this part, is this meant to address the shared checker workflow in the prod db?
But this seems to delete all `checkerid` in these tables. Could be code comment

Reply:
Was here, waiting to see the answer to Denis' question above.

Reply:
Answered in after stand-up chat, needs a bit of rework